Post-OT updates to the PBS article (#2367)
GoogleChrome/web.dev: src/site/content/en/blog/periodic-background-sync/index.md

  syncs. Retrieve an instance of this class from the ServiceWorkerRegistration.periodicSync`
  property.
* `ServiceWorkerGlobalScope.onperiodicsync`. Registers a handler to receive the `PeriodicSyncEvent`.

> * `ServiceWorkerRegistration.periodicSync`. Holds a reference to the `PeriodicSyncManager`.

Reviewer: ```suggestion
* `ServiceWorkerRegistration.periodicSync`. Returns a reference to the `PeriodicSyncManager`.
```

Reviewer: I guess 'holds' would be more logical, but 'returns' is [what's typically done](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration#Properties). Language follows convention more than it does strict logic.

Author: 'returns' implies that it's a function rather than a property to me, but I'm happy to go with convention here.

Reviewer: I think you should. 

This is why I'm so strict about including `()` at the end of function names.

});
```

### Adding periodic background sync to an existing web app

> [This set of changes](https://github.com/GoogleChromeLabs/so-pwa/pull/11) were needed to add in

Reviewer: ```suggestion
[This set of changes](https://github.com/GoogleChromeLabs/so-pwa/pull/11) were needed to add
```

periodic background sync to an [existing web app](https://so-pwa.firebaseapp.com/) that already had

> offline support. This example includes a number of helpful logging statements that describe the

Reviewer: ```suggestion
periodic background sync to an [existing PWA](https://so-pwa.firebaseapp.com/). This example includes a number of helpful logging statements that describe the
```

Reviewer: Unless I'm missing something. 

What am I looking for at the end of that link? If you can't link to a specific question, can you tell the reader what they're looking for?

Author: Your change seems 👍
